packages/synapse-react-client/src/components/RssFeedCards/RssFeedCards.tsx

```diff
@@ -117,13 +117,12 @@ export class RssFeedCards extends React.Component<RssFeedCardsProps, RssState> {
             {this.state.rssFeed.items &&
               this.state.rssFeed.items.map((item: any, index: any) => {
                 // The other is to hide the large number of items in a particular feed (usually a max of 10 are returned).  See state.isShowingMoreItems
-                const isItemVisible: boolean = index < this.props.itemsToShow
+                if (index >= this.props.itemsToShow) {
+                  return
+                }
 
                 return (
-                  <div
-                    key={item.guid}
-                    className={`FeedItem ${isItemVisible ? '' : 'hidden'}`}
```

Comment:
I think the '.hidden' css class has been removed, because all items with this class were still being shown in the portal.  Changed to just not include in the return (in the DOM!)